File: URI/concluidos/Javascript/1012.js
var input = require('fs').readFileSync('./dev/stdin/arquivo.txt', 'utf8');
var lines = input.split('\n');

const pegarValores = (line) => line.split(" ").map(a => +a);
const [a, b, c] = pegarValores(lines.shift());
var pi = 3.14159;

console.log(`TRIANGULO: ${((a*c)/2).toFixed(3)}`);
console.log(`CIRCULO: ${(pi * Math.pow(c,2)).toFixed(3)}`);
console.log(`TRAPEZIO: ${(((a+b)*c)/2).toFixed(3)}`);
console.log(`QUADRADO: ${(Math.pow(b,2)).toFixed(3)}`);
console.log(`RETANGULO: ${(a*b).toFixed(3)}`);

File: URI/concluidos/Python/1012.py
import math
line = input().split(" ")
a, b, c = line

print('TRIANGULO: %0.3f' % ((float(a) * float(c))/2))
print('CIRCULO: %0.3f' % (math.pow(float(c),2) * 3.14159))
print('TRAPEZIO: %0.3f' % (((float(a) + float(b)) * float(c)) / 2))
print('QUADRADO: %0.3f' % (math.pow(float(b), 2)))
print('RETANGULO: %0.3f' % (float(a) * float(b)))